src/pages/InitMsig.vue

```diff
@@ -214,63 +223,77 @@ export default defineComponent({
 			errorMessage(this.key);
 			openErrorNotificationWithIcon(error.message);
 		},
+		createMultisig() {
+			let sender = algosdk.decodeUnsignedTransaction(
+				convertBase64ToUnit8Array(this.unsignedInput)
+			);
+			let fromAddr = algosdk.encodeAddress(sender.from.publicKey);
+			let addr = this.addresses.map((x) => {
+				return x.address;
+			});
+
+			let version = this.version * 1;
```

Comment:
why do we multiply by 1 here?

Reply:
if its undefined, it throws an error then (if user hasn't input any value)

Reply:
then why not use `this.version ?? 1` ?

Reply:
?

Reply:
Why can't we simply typecast? eg. `Number(this.version)`?

Reply:
updated